rllib/agents/dqn/r2d2.py

```diff
@@ -31,15 +31,8 @@
 
         # === Replay buffer ===
         "replay_buffer_config": {
-            # For now we don't use the new ReplayBuffer API here
-            "_enable_replay_buffer_api": False,
+            "_enable_replay_buffer_api": True,
             "type": "MultiAgentReplayBuffer",
-            "prioritized_replay": False,
```

Comment:
`prioritized_replay`: We don't support this anymore, as specification of replay buffers is not explicit.
Meaning that If you want to use a prioritized replay buffer, you have to put a Buffer that allowes PER in the `type` field.

The other prioritization args are allowed, but since R2D2 does not use PER by default, the keys are not used.
And if user specifies a new buffer type, he/she can either specify these args explicitly, or they will not be part of the buffer config, which is not an issue because the values we see here are simply the default values of the paper which are also our default values.

Reply:
Got it. I thought R2D2 did use PR, but that's cool, then. Users can always specify type="MultiAgentPrioritizedReplayBuffer".